components/Base/Banner.js

```diff
@@ -8,16 +8,27 @@ const HeaderStyle = {
     fontSize: 48
 }
 
-export default ({title}) => (
-<Segment className="banner-content" vertical style={{padding: 0, borderBottom: 0}}>
-    <Rail size='massive' internal position='left' style={{}}>
-        <Segment vertical style={{padding: 0, borderBottom: 0, position: 'absolute', left: '10vw',  bottom: 0, width: 400, zIndex: 2 }}>
-            <Image fluid src='/static/images/faqs/shell.png' />
-            <Rail size='massive' internal position='left' style={{paddingLeft: 120, top: 60}}>
-                <Segment basic><Header style={HeaderStyle}>{title}</Header></Segment>                
-            </Rail>
-        </Segment>
-    </Rail>
-    <WavyBannerSvg />
-</Segment>
-)
+export default ({title, imagePath}) => {
+
+    const BannerContentStyled = {
+        position: 'relative',
+        minHeight: '30rem !important',
+        backgroundImage: `url(${ imagePath ? imagePath : '/static/images/faqs/banner.png' } ) !important`,
```

Comment:
We may need to fix this in the future as it will be carousel and not a background image

Reply:
this is a each banner in each page? should have only 1 banner for that story,
if use carousel we must create new one

Reply:
that is why i wanted to keep from the comment above. it already supports `<Image />`. In the future we can replace `<Image />` with maybe `<Carousel />`. Or maybe we can come up with something new

Reply:
Okie, so create new one for Carousel
For this I use image because need to set position of image is origin is center and support many image size, that my point

Reply:
Its cool bruh